srcpkgs/adobe-flex-sdk/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'adobe-flex-sdk'
+pkgname=adobe-flex-sdk
+version=4.6
+revision=1
+create_wrksrc=yes
+depends="openjdk-jre"
+short_desc="Freely develop and deploy Flex applications"
+maintainer="fosslinux <fosslinux@aussies.space>"
+license="custom:Adobe License (non free)"
+homepage="https://www.adobe.com/devnet/flex/flex-sdk-download.html"
+distfiles="http://download.macromedia.com/pub/flex/sdk/flex_sdk_${version}.zip"
+checksum=622b63f29de44600ff8d4231174a70fcb3085812c0e146a42e91877ca8b46798
+repository="nonfree"
+restricted=yes
+# free(): invalid pointer
+nostrip=yes
```

Comment:
Make it clear that `nostrip` is because of this error, if that's the case.

Reply:
How do I do that better than in a comment above it?

Reply:
Yeah, basically. When first looking at it, it looked like you had pasted something onto it. Either your example or `stripping causes error frere(): invalid pointer` should be good.